### ISSUE TITLE
Reexport MonadError

### ISSUE THREAD

**Issue by nikita-volkov** [Gabriel439/Haskell-Errors-Library#21]
http://hackage.haskell.org/package/mtl-2.1.2/docs/Control-Monad-Error-Class.html#t:MonadError

**Comment by cideM** [comment]
Sorry for resurrecting this from the dead but I'm struggling to use the many helpful utility functions from `errors` in my application that uses a very standard `mtl` architecture.

I'd like to keep the convenience of just adding a type class to a function, but also map over `Left` and use the utility functions from `Control.Error`. But `MonadError` appears to be the only `mtl` type class and that doesn't work with `Control.Error`

**Comment by Gabriel439** [comment]
@cideM: Yeah, I'm not aware of a good solution to that problem using `mtl`
